JoinExample.sql

```diff
@@ -0,0 +1,52 @@
+USE BankDB;
```

Comment:
This is no different then c# in formatting matter. Please format the document, add spaces where needed and remove extra spaces/lines